README.md

```diff
@@ -336,6 +337,20 @@ To avoid having to run the databse setup for each test run you can specify the `
     
 See `corehq.tests.nose.HqdbContext` for full description of `REUSE_DB`.
 
+### Running tests by tag
+You can run all tests with a certain tag as follows:
+
+    $ ./manage.py test --attr=tag
+
+Available tags:
+
+  * dual_backend: all tests decorated with `run_with_all_backeds`
```

Comment:
nit: maybe rename this to `all_backends` to be consistent with the decorator name and in case we ever have more than two backends?